src/legacy/config/config-set.spec.ts

```diff
@@ -918,7 +917,6 @@ describe('_resolveTsConfig', () => {
 
         const conf = cs.parsedTsConfig
         expect(conf.options.path).toBe(tscfgPathStub)
-        expect(findConfig).not.toHaveBeenCalled()
```

Comment:
this will now be called to find out if we there are project references in the tsconfig to check.